### ISSUE TITLE
Ruby feedback for `for` loop refers to non-existant method.

### ISSUE THREAD

**Issue by kotp** [exercism/rikki#14]
There is no `Enumerable#each` method.

![rikki_enumerable_vs_for_comment](https://cloud.githubusercontent.com/assets/16747/14605085/f6d821c4-0544-11e6-9c0b-a6a3bac6ce1a.png)

There is a `map` method though, as you can see below.. 
``` ruby
>> Enumerable.instance_methods.include? :each
=> false
>> Enumerable.instance_methods.include? :map                                       
=> true
```

The documentation has this to say about `each`.

```markdown
= Enumerable

(from ruby site)
------------------------------------------------------------------------------
The Enumerable mixin provides collection classes with several traversal and
searching methods, and with the ability to sort. **The class must provide a
method each**, which yields successive members of the collection. If
Enumerable#max, #min, or #sort is used, the objects in the collection must
also implement a meaningful <=> operator, as these methods rely on an ordering
between members of the collection.

------------------------------------------------------------------------------
```

Note: emphasis is mine.

**Comment by kotp** [comment]
Perhaps we don't mention Enumerable here at all, but do show the observer method `each` that takes a variable, modifies it, and shows that the outside variable survives.

Something like this:

```ruby
people = ["Alice", "Bob", "Charlie"]
person = "Alice"
people.each do |person| # Note the duplicate name of the "person" variable outside this block
  person = person.length # Note the changes to the person variable here.
end
person #=> "Alice"
```
Notice that the variable `person` survives even modification inside the `each` block.

**Comment by kytrinyx** [comment]
For some reason I always think of `each` as being `Enumerable`, even though I always have to implement it myself when I mix Enumerable in. Stupid brain.

I suppose getting people to switch from `for` to `each` is a great step, and then once people are using `each` we can suggest that they look at other options if they've nested conditionals inside the block, or if they're using temporary variables.

**Comment by Insti** [comment]
This issue was resolved by #15 and can be closed.

**Comment by kotp** [comment]
Thanks @Insti
